data/report_templates/project_summary.md

```diff
@@ -17,6 +17,11 @@ NGI Project Name
 NGI Project ID
 :   {{ project.ngi_id }}
 
+{% if project.ngi_facility %}
```

Comment:
Ah nice! Well, if you fancy going through the IGN report template and adding some hyphens + line breaks to improve readability, that would be much appreciated :wink: